src/main/java/com/db/db/volunteers/controller/VolunteerController.java

```diff
@@ -33,6 +33,29 @@ public VolunteerController(VolunteerService volunteerService, GroupService group
         this.pollingUnitService=pollingUnitService;
     }
 
+    @GetMapping("/pollingunitsimple")
+    public ModelAndView pollingUnits(Form form, ModelAndView mv){
+        mv.setViewName("pollingunitsimple");
+        form.setPollingUnits(volunteerService.listPollingUnitWithNoVolunteer());
+        form.setPollingUnitVolunteerStats(volunteerService.listPollingUnitVolunteerStats());
+        return mv;
+    }
+/*
+    @GetMapping("/pollingunits")
```

Comment:
There is quite a few commented code.  Obviously, there's a plan to implement a more **robust version of Polling Unit listing** by Volunteer count.  This should be done ASAP and not be left to linger.